ecommerce/filters.py

```diff
@@ -15,3 +16,32 @@ class ProductFilter(filters.FilterSet):
     class Meta:
         model = Product
         fields = []
+
+
+class CouponUtils:
```

Comment:
I think the better place for these utils is ecommerce/utils.py. filters.py is meant to be for the API filters that we create using django_filters.